test/004.test.js

```diff
@@ -1,167 +0,0 @@
-/* eslint-disable no-unused-expressions */
-/* eslint-disable no-undef */
-import * as Factory from './lib/factory.js';
-chai.use(chaiAsPromised);
-const expect = chai.expect;
-
-describe('004 protocol operations', function () {
```

Comment:
Where is this file in the new setup?

Reply:
See https://github.com/standardnotes/snjs/blob/migrate-chai-tests/packages/snjs/test/lib/004.encryption.test.ts